courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2007 @@
+{
```

Comment:
<p>Same as above, I'd move these comments to a single markdown cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Created a markdown cell with the text:</p><p> </p><p>Again, we can use .describe() to see some summary statistics for the numeric fields in our dataframe.&nbsp;The count shows 4087.000000 for all feature columns. Thus, there are no missing values.</p>